php/Terminus/Models/Upstream.php

```diff
@@ -27,6 +27,14 @@ public function __construct($attributes, array $options = [])
         }
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function __toString()
+    {
+        return "{$this->site->id}: {$this->url}";
```

Comment:
This should be $this->id. Are you having any issues where the Upstream ID is not being passed through?